### DISCUSSION TITLE
전체적인 색깔 변경에 대한 문의입니다.

### DISCUSSION THREAD

**Discussion by daeunni** [pronist/hELLO#100] [Q&A] [answered]
안녕하세요! 좋은 스킨으로 도움 주셔서 감사합니다. :) 

다름이 아니라 혹시 스킨의 메인 칼라가 black으로 설정되어 있는 것 같습니다. (위 스크롤이나 하트 버튼 등) 
이런 색상을 혹시 변경할 수 있는 부분이 있을까요? 

감사합니다.

**Comment by pronist** [comment, marked as answer]
안녕하세요 :) [기여하기](https://github.com/tistory-projects/tistory-skin-hELLO/blob/master/CONTRIBUTING.md)에 가시면 버튼 및 배경과 같은 요소에 대해 색상을 지정할 수 있는 CSS 변수를 살펴보실 수 있습니다. 이 부분을 수정하시면 됩니다. 그 외에 스킨 옵션 등으로 색상을 변경하는 것은 제공하지 않습니다.

**Reply by daeunni** [reply]
친절한 답변 감사드립니다 :)
